CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### Unreleased
   - Validate ids based on schema
   - Validate data for different survey types
+  - SUpport 'completed' flag in schema definitions
```

Comment:
May actually be worth linking to the schema definition (maybe somewhere in here? https://github.com/ONSdigital/ons-schema-definitions/tree/master/docs)